src/App/WarehouseService.cs

```diff
@@ -3,25 +3,45 @@
 
 public class WarehouseService
 {
-    private readonly string _connectionString = "Host=localhost;Username=postgres;Password=yourpassword;Database=warehouse";
+    private readonly string _connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING");
 
-    public void SaveData(string key, string value)
+    public void SaveData(string partName, int bikeTypeId, int stock)
     {
+        try
```

Comment:
make the exception handling more granular or make it catch specific exceptions, right now it basically just says, damn i broke. And more specific error handling on the connection or on the execution, which have their own exception types, can help locate bugs easier